catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -459,9 +464,17 @@ private Path getTempContentItemDir(String requestId, URI contentUri) {
     private Path getContentItemDir(URI contentUri) {
         List<String> pathParts = getContentFilePathParts(contentUri.getSchemeSpecificPart(),
                 contentUri.getFragment());
-
-        return Paths.get(baseContentDirectory.toAbsolutePath()
-                .toString(), pathParts.toArray(new String[pathParts.size()]));
+        try {
+            return Paths.get(baseContentDirectory.toString(),
+                    pathParts.toArray(new String[pathParts.size()]));
+        } catch (InvalidPathException e) {
+            LOGGER.debug("Invalid path: [{}/{}]",
+                    baseContentDirectory.toAbsolutePath()
```

Comment:
I know it's just debug logging but  could still use canonical path here to be consistent. 😄